build_tools/waf_dynamo.py

```diff
@@ -351,11 +353,11 @@ def default_flags(self):
         os.makedirs(libpath)
     self.env.append_value('LIBPATH', libpath)
 
-    self.env.append_value('INCLUDES', build_util.get_dynamo_ext('include'))
     self.env.append_value('INCLUDES', build_util.get_dynamo_home('sdk','include'))
-    self.env.append_value('INCLUDES', build_util.get_dynamo_home('include'))
     self.env.append_value('INCLUDES', build_util.get_dynamo_home('include', build_util.get_target_platform()))
+    self.env.append_value('INCLUDES', build_util.get_dynamo_home('include'))
     self.env.append_value('INCLUDES', build_util.get_dynamo_ext('include', build_util.get_target_platform()))
+    self.env.append_value('INCLUDES', build_util.get_dynamo_ext('include'))
```

Comment:
Fixing the include order by priority:

includes/<platform>
includes/
ext/includes/<platform>
ext/includes/